### PR TITLE
Update careers.json

### PR DESCRIPTION
**Pull request by saracmcorreia**

### CODE REVIEW

**Comment by ibrennan** [comment]
Will review when the other roles have been added @saracmcorreia

File: _data/careers.json [hunk 1]
```diff
@@ -45,4 +45,18 @@
 		"url": "https://fathom-london.breezy.hr/p/562174f432a4-senior-java-engineer-contract",
 		"open": false
 	}
+	{
+		"title": "IA/UX Designer (Contract)",
+		"description":
+			"We are looking to hire a brilliant IA/UX Designer to join our talented team. We want smart, passionate practitioners who are ambitious to make an impact with their solutions whilst working with one of the best teams in London.",
+		"url": "https://fathom-london.breezy.hr/p/c892b0801e35-ia-ux-designer-contract",
+		"open": true
+	}
```
Comment (ibrennan) [inline _data/careers.json]: Missing a comma here

File: _data/careers.json [hunk 2]
```diff
@@ -45,4 +45,18 @@
 		"url": "https://fathom-london.breezy.hr/p/562174f432a4-senior-java-engineer-contract",
 		"open": false
 	}
```
Comment (ibrennan) [inline _data/careers.json]: Missing a comma here

**Comment by peduarte** [comment]
👍

**Pull request merged**